versioned_sidebars/version-3.9-sidebars.json

```
@@ -197,51 +437,682 @@
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/getting-started-monitoring",
+                  "label": "Helm Chart を使用した監視用 Prometheus Operator"
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/getting-started-logging",
+                  "label": "Helm Chart を使用したログ記録用 Loki スタック"
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/getting-started-scalar-manager",
+                  "label": "Helm Chart を使用した Scalar Manager"
+                }
+              ]
+            },
+            {
+              "type": "category",
+              "label": "カスタム値を設定",
+              "collapsible": true,
+              "items": [
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-scalardl-ledger",
+                  "label": "ScalarDL Ledger のカスタム値ファイルを構成"
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-scalardl-auditor",
+                  "label": "ScalarDL Auditor のカスタム値ファイルを構成"
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-scalardl-schema-loader",
+                  "label": "ScalarDL Schema Loader のカスタム値ファイルを構成"
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-scalar-admin-for-kubernetes",
+                  "label": "Scalar Admin for Kubernetes のカスタム値ファイルを構成"
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-scalar-manager",
+                  "label": "Scalar Manager のカスタム値ファイルを構成"
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-envoy",
+                  "label": "Scalar Envoy のカスタム値ファイルを構成"
+                }
+              ]
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/how-to-deploy-scalar-products",
+              "label": "Helm Chart を使用して Scalar 製品をデプロイ"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/how-to-deploy-scalardl-ledger",
+              "label": "Helm Chart を使用して ScalarDL Ledger をデプロイ"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/how-to-deploy-scalardl-auditor",
+              "label": "Helm Chart を使用して ScalarDL Auditor をデプロイ"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/how-to-deploy-scalar-admin-for-kubernetes",
+              "label": "Kubernetes 用の Scalar Admin をデプロイ"
```

```suggestion
              "label": "Helm Chart を使用して Scalar Admin for Kubernetes をデプロイ"
```
Same as `sidebars.js`.